resources-domain/packages/resources-search-config/getCreateIndexParams.ts

```diff
@@ -40,28 +34,7 @@ import {
  * @param index
  */
 export const getCreateIndexParams = (index: string): IndicesCreateRequest => {
-  const { mappingFields, languageFields } = resourceIndexDocumentMappings;
-
-  const convertMappingFieldsToProperties = () =>
```

Comment:
This function seemed to better fit into the resourceIndexDocumentMappings library.

I moved it there and did a bit of refactoring. See notes below